Integrations/python/deephaven/TableManipulation/__init__.py

```diff
@@ -8,13 +8,15 @@
 
 import jpy
 
-__all__ = ['ColumnRenderersBuilder', 'DistinctFormatter', 'DownsampledWhereFilter', 'LayoutHintBuilder',
-           'SmartKey', 'SortPair', 'TotalsTableBuilder', 'WindowCheck']
+__all__ = ['ColumnRenderersBuilder', 'DBDateTime', 'DistinctFormatter', 'DownsampledWhereFilter', 'DynamicTableWriter', 
+           'LayoutHintBuilder', 'SmartKey', 'SortPair', 'TotalsTableBuilder', 'WindowCheck']
 
 # None until the first successful _defineSymbols() call
 ColumnRenderersBuilder = None   #: Class to build and parse the directive for Table.COLUMN_RENDERERS_ATTRIBUTE (io.deephaven.db.v2.ColumnRenderersBuilder).
 DistinctFormatter = None        #: Class to create distinct and unique coloration for each unique input value (io.deephaven.db.util.DBColorUtil$DistinctFormatter).
+DBDateTime = None               #: Class to create a DBDateTime instance in Python (io.deephaven.db.tables.utils.DBDateTime).
```

Comment:
This doesn't seem to belong here.  This module is documented to be `Functionality to display and modify tables.`  I don't see how a date time does that.  It makes far more sense in `Types`.